File: Modul 6/Bahasa C (modul 6)/PRAK604-2310817320015-AlfisyahRinaFajriati.c
#include <stdio.h>
#include <string.h>

void main(){

    char kode[100], pesan[100] ;
    int karakter1, karakter2, i, bintang = 0, pagar = 0 ;
    scanf ("%[^\n]%*c", &kode) ;
    scanf ("%[^\n]%*c", &pesan) ;
    karakter1 = strlen (kode) ;
    karakter2 = strlen (pesan) ;
    if (karakter1 != karakter2) {
        printf ("Panjang kalimat berbeda, pesan palsu") ; }
    else {
        for (i = 0 ; i < karakter1 ; i++) {
            if (kode[i] == pesan[i]) {
                if (kode[i] == ' ') {
                    printf (" ") ; }
                else {
                    printf ("*") ;
                    bintang++ ; } }
            else {
                printf ("#") ;
                pagar++ ; } }
        printf ("\n* = %d", bintang) ;
        printf ("\n# = %d", pagar) ;
        if (bintang >= pagar) {
            printf ("\nPesan Asli\n") ; }
        else {
            printf ("\nPesan Palsu\n") ; } }
}

File: Modul 6/Bahasa Python (modul 6)/PRAK604-2310817320015-AlfisyahRinaFajriati.py
kode = input() ; pesan = input()
karakter1 = len(kode) ; karakter2 = len(pesan)
bintang = 0 ; pagar = 0
if karakter1 != karakter2 :
    print("Panjang kalimat berbeda, pesan palsu")
else :
    for i in range(0, karakter1) :
        if kode[i] == pesan[i] :
            if kode[i] == ' ' :
                print(" ", end='')
            else :
                print("*", end='') ; bintang += 1
        else :
            print("#", end='') ; pagar += 1
    print("\n* =", bintang)
    print("# =", pagar)
    if bintang >= pagar :
        print("Pesan Asli")
    else :
        print("Pesan Palsu")